packages/terra-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -174,7 +174,14 @@ Terra.describeViewports('WorklistDataGrid', ['medium', 'large'], () => {
       rowSelectionNavigateToCell(4, 1);
       browser.keys(['Escape']); // clears row selection mode
 
-      Terra.validates.element('row-selection-mode-off-cell-4-1-focused', { selector: defaultSelector });
+      Terra.validates.element('row-selection-mode-off-cell-4-0-focused', { selector: defaultSelector });
+    });
+
+    it('clears row selection mode when focus is on a row selection cell', () => {
```

Comment:
This reads as "row selection mode is cleared due to receiving focus on a row selection cell". Based on the PR description, I would suggest:

```suggestion
    it('does not change focus when row selection mode is cleared when focused on a row selection cell', () => {
```

I would also suggest asserting that focus is on the right element.